bundles/binding/org.openhab.binding.rcswitch/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,32 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.binding.rcswitch.internal
+Ignore-Package: org.openhab.binding.rcswitch.internal
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Name: openHAB RC Switch Binding
```

Comment:
rename binding to something more specific like "rpircswitch” "openHAB Raspberry Pi RC Switch Binding"?